BrainPortal/spec/boutiques/validator.rb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/ruby
```

Comment:
See comment about validator in general conversation: shall we use a git submodule? https://git-scm.com/docs/git-submodule